server/controllers/notificationController.js

```diff
@@ -91,29 +118,58 @@ const notificationController = {
           },
         }
       )
-      .then(() => res.redirect('/api/v1/notifications'))
-      .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the serve'));
+      .then(() => res.status(200).json({
+        status: 'success',
+        message: 'All notifications have been marked as read'
+      }))
+      .catch(() => res.status(500).jsend.error('Oops, something has gone wrong on the server'));
   },
   clearHistory: (req, res) => {
-    Notifications
+    Notifications.scope(null)
```

Comment:
why do we have to call the `.scope()` method here

Reply:
The request doesn't run without scope. That's why the scope is null.